components/automate-ui/src/app/modules/infra-proxy/select-box/src/lib/select-box.component.scss

```diff
@@ -31,14 +31,20 @@
 .run-list-box {
   display: flex;
   padding: 10px 10px;
-  color: rgba(0, 0, 0, 0.87);
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
   box-sizing: border-box;
   cursor: move;
   background: $chef-white;
   font-size: 14px;
+  margin-top: 6.5px;
+}
+
+// sass-lint:disable no-important
+::ng-deep .cdk-drag-preview {
+  position: relative;
+  z-index: 999999999 !important;
```

Comment:
this can lead to problems down the line - lets first see if we can use something other than `999999999` and we generally do not allows the !important flag in our codebase

Reply:
yes, I have changed the z-index to this 999999 but if I try to remove the ```! important``` the dragged property will go behind the pop-up. Like this: 

https://user-images.githubusercontent.com/61003053/115260542-7b1ebe00-a150-11eb-9302-75fe9b73b09f.mp4